packages/components/src/vue/components/word-list/word-item-panel.vue

```diff
@@ -127,6 +130,9 @@ export default {
     },
     setLemmaFilterByClick(lemma) {
       this.$emit('setLemmaFilterByClick', lemma)
+    },
+    formattedWord (lemma) {
+      return Lemma.getDisplayWord(lemma)
```

Comment:
And the same here (my question is in the next note) - I don't understand why do we need these additional code.
From my point of biew it is  a commonly used getter for a specific instance of Lemma, but not an abstarct formatter of any String.

Reply:
We're using `displayWord` to get the lemma's word formatted for the display in the popup. We need to apply the same rules to display lemma words in the wordlist (@balmas please correct me if I'm wrong). However, we do not have `Lemma` objects in the wordlist, lemma words are presented as strings there. Because of that, using a Lemma's getter is not an option. So I came up with with a static method that can be used to format a string representing a lemma's word. It's not supposed to be used with any string, but only with those that represent lemma words. I agree it's not ideal but I don't see a better solution now.

I believe it's more than just removing the trailing digits, its to make lemmas in the wordlist be formatted the same way as in other places of the app, so I think we should use the same formatting routine for both.

Do you have any suggestion how to do it differently, considering a limited amount of time that can be spent on this?

Reply:
In the `word-item-panel.vue` we use 
```
lemmasList () {
  return this.$store.state.app.wordListUpdateTime ? this.worditem.lemmasList.split(',') : []
}
```
in the wordItem class we could see that
```
  get lemmasList () {
    if (this.homonym && this.homonym.lexemes) {
      return this.homonym.lexemes.map(lexeme => lexeme.lemma.word).filter((value, index, self) => {
        return self.indexOf(value) === index
      }).join(', ')
    }
    return ''
  }
 ```
 
 So this way you could see, that I used lemmas but not some simple words.
In this case I suggest to create `displayLemmaList` - and use `lemma.displayWord` instead of `lemma.word`
 
 I think it would be more consistent
 
 Hope it will help

Reply:
I thought of using `Lemma` objects inside the word item Vue component but decided it would be too complex because the changes will have the ripple effect. But if we create a `Lemma` object inside the `formattedWord()` method then all the changes would be isolated there. That would allow to keep it simple and yet avoid adding a static method to the `Lemma`. So that's what I did. Please let me know what do you think.

Reply:
I didn't understand, why you didn't decide to simply add a new display method for lemmasList inside `data-models` WordItem` class.
It could be used in other places in that case.
From my point of view - your decision is more complex.

But if you think that your solution is better , then you could do as you think.

Reply:
> I didn't understand, why you didn't decide to simply add a new display method for lemmasList inside data-models WordItem` class.
It could be used in other places in that case.

I was not sure if it would be useful in other places, but that's maybe because I don't have all the full picture of all the usages of the wordlist. The current fix keep changes rather minimal and I think it might be good this way. We're planing to refactor wordlist later so we can postpone the other changes till then.

Reply:
It is also used inside download of the wordlist
`download-confirmation.vue`

If you apply your solution - then you would fix inside the wordlist panel, but download would be with digits.

That's why - I suggest to create only one getter and used in both places

Reply:
> If you apply your solution - then you would fix inside the wordlist panel, but download would be with digits.

If I understand correctly, we need to remove digits from the display only. We would still want to keep digits in all our data structures internally. @balmas, is that correct?

Reply:
yes that's correct.